uisdk/src/main/java/com/karhoo/uisdk/screen/booking/BookingActivity.kt

```diff
@@ -197,23 +196,6 @@ class BookingActivity : BaseActivity(), AddressBarMVP.Actions, BookingMapMVP.Act
         }
     }
 
-    override fun onRequestPermissionsResult(requestCode: Int,
```

Comment:
How do we handle this now? 🤔

Reply:
A good question! Need to look into why this isn't being used. 🤔

Reply:
I've put it back in for now but we need to look at why it doesn't seem to be used.